SurveyApp.DAL/Repositories/SurveyRepository.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Authentication;
+using System.Threading.Tasks;
+using SurveyApp.DAL.DataContext;
+using SurveyApp.DAL.EntityModels;
+using SurveyApp.DAL.Repositories.Interfaces;
+
+namespace SurveyApp.DAL.Repositories
+{
+    public class SurveyRepository : Repository<SurveyDataModel>, ISurveyRepository
+    {
+        public SurveyRepository(ApplicationContext context) : base(context)
+        {
+        }
+
+        public async void SaveAsync(SurveyDataModel surveyToSave)
```

Comment:
This method is almost the same as in SurveyTemplate repository. Consider inheritance to prevent duplications.